.travis.yml

```diff
@@ -4,58 +4,38 @@ branches:
 os: osx
 matrix:
   include:
-    # Note: The ordering of the environments is optimized so that the complete matrix runs as quickly as possible,
-    # while also providing the first results as early as possible.
```

Comment:
I'm very glad that our new CI configuration has better build times so we no longer have to do this sort of manual optimization. :)